app/bundles/LeadBundle/Form/Type/CustomYesNoButtonGroupType.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
```

Comment:
Declare strict types on all new classes
```suggestion

declare(strict_types=1);

```